src/utils/pause/scopes/translateScope.js

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
+
+// @flow
+import { getBindingVariables, getSourceBindingVariables } from "./getVariables";
+import { getFramePopVariables, getThisVariable } from "./utils";
+import { simplifyDisplayName } from "../../frame";
+
+import type {
+  Frame,
+  Pause,
+  Scope,
+  SyntheticScope,
+  BindingContents
+} from "debugger-html";
+
+import type { NamedValue } from "./types";
+
+function getScopeTitle(type, scope) {
+  if (type === "function") {
+    return scope.function.displayName
+      ? simplifyDisplayName(scope.function.displayName)
+      : L10N.getStr("anonymous");
+  }
+  return L10N.getStr("scopes.block");
+}
+
+export function translateScope(
```

Comment:
does the name translate mean anything special?

would `getScope`, which is parallel to `getScopes` work?